<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlPage</class>
 <widget class="QMainWindow" name="HtmlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="go">
    <property name="title">
     <string>برو به</string>
    </property>
    <addaction name="search_kms"/>
    <addaction name="add_kms"/>
   </widget>
   <widget class="QMenu" name="user">
    <property name="title">
     <string>کاربر</string>
    </property>
    <addaction name="logout"/>
    <addaction name="separator"/>
    <addaction name="add_user"/>
   </widget>
   <addaction name="go"/>
   <addaction name="user"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="search_kms">
   <property name="text">
    <string>جستجوی مورد دانشی</string>
   </property>
  </action>
  <action name="add_kms">
   <property name="text">
    <string>افزودن مورد دانشی</string>
   </property>
  </action>
  <action name="logout">
   <property name="text">
    <string>خروج</string>
   </property>
  </action>
  <action name="add_user">
   <property name="text">
    <string>افزودن کاربر</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
